Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -239,7 +240,7 @@ PlayerSettings:
   clonedFromGUID: 56e7a2d3a00f33d44bdd161b773c35b5
   templatePackageId: com.unity.template.3d@1.0.0
   templateDefaultScene: Assets/Scenes/SampleScene.unity
-  useCustomMainManifest: 0
+  useCustomMainManifest: 1
```

Comment:
I had to change to use a custom AndroidManifest in Subtester in order to add the required entry to the AndroidManifest.